src/server/scripts/OutdoorPvP/OutdoorPvPNA.cpp

```diff
@@ -16,6 +16,7 @@
  */
 
 #include "OutdoorPvPNA.h"
+#include "GridNotifiers.h"
```

Comment:
#include "ScriptedCreature.h"

Reply:
Just saw the error and i just added now xDD haha

Reply:
Can't forget the includes :D

Reply:
VS is nice because sometimes it just dont care about includes. But any other compiler actually cares

Reply:
Im used to code in Java, import are done automaticly D:

Reply:
Welcome to C++ :D